### DISCUSSION TITLE
Failed to locate rsgain. Enter the path in plugin settings (Mac)

### DISCUSSION THREAD

**Discussion by oranjebloom** [complexlogic/rsgain#83] [Q&A] [answered]
Hi, I have successfully installed rsgain on Mac via Homebrew and have been using it via the command line, but I wanted to also use it in MusicBrains Picard which I installed just today after having previously installed rsgain. However, when I try to calculate replaygain I get the message "Failed to locate rsgain. Enter the path in plugin settings". Can anyone tell me how I can locate rsgain in the settings so that I can get MusicBrains Picard to work? I have no idea where rsgain is located on the system.

**Comment by complexlogic** [comment, marked as answer]
In your terminal, type `which rsgain`. Copy the result, and paste in the Picard plugin setting "Path to rsgain"

**Reply by oranjebloom** [reply]
Thank you. Your help solved the issue.
